syntax = "proto3";

service Greeting {
  // Sends a greeting
  rpc SayHello (HelloRequest) returns (HelloReply) {}
  rpc SayHelloAgain (HelloRequest) returns (HelloReply) {}
  rpc SendFile (File) returns (HelloReply){}
}

message File{
	string fileName = 1;
	string fileContent = 2;
	}
	
// The request message containing the user's name.
message HelloRequest {
  string name = 1;
}

// The response message containing the greetings
message HelloReply {
  string message = 1;
}
